morse.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'hash'
+
+def decode_char(char)
+  HASH[char]
+end
+
+def decode_word(word)
+  new_word = ''
+  word.split.each do |char|
+    new_word += decode_char(char)
+  end
+  new_word
+end
+
+def decode_message(message)
+  new_message = ''
+  message.split('  ').each do |word|
+    new_message += "#{decode_word(word)} "
+  end
+  new_message
+end
```

Comment:
- [ ] [OPTIONAL] I noticed the `decode_message` function is adding unnecessary spaces between words and after the last word as shown below so please consider fixing this issue.
- Here's how it looks:
  ![image](https://github.com/danifromecuador/decode-morse-code/assets/104968236/66ace67a-a9bd-4766-8292-d038bd60bbbc)

Reply:
Thanks, I've fixed that issue using map instead each method.